client/my-sites/plugins/main.jsx

```diff
@@ -442,25 +454,78 @@ export class PluginsMain extends Component {
 						</div>
 					</FixedNavigationHeader>
 				) }
-				<div className="plugins__main">
-					<div className="plugins__main-header">
-						<SectionNav selectedText={ this.getSelectedText() }>
-							<NavTabs>{ navItems }</NavTabs>
-							<Search
-								pinned
-								fitsContainer
-								onSearch={ this.props.doSearch }
-								initialValue={ this.props.search }
-								ref={ `url-search` }
-								analyticsGroup="Plugins"
-								placeholder={ this.getSearchPlaceholder() }
-							/>
-						</SectionNav>
+				<div className={ classNames( { 'plugins__top-container': isJetpackCloud } ) }>
+					<div
+						className={ classNames( {
+							'plugins__content-wrapper': isJetpackCloud,
+						} ) }
+					>
+						{ isJetpackCloud && (
+							<div className="plugins__page-title-container">
+								<h2 className="plugins__page-title">{ pageTitle }</h2>
+								<div className="plugins__page-subtitle">
+									{ this.props.selectedSite
+										? this.props.translate( 'Manage all plugins installed on %(selectedSite)s', {
+												args: {
+													selectedSite: this.props.selectedSite.domain,
+												},
+										  } )
+										: this.props.translate( 'All sites' ) }
```

Comment:
What do you think about changing `All sites` here to `Manage plugins installed on all sites`?

Reply:
Yes, sounds good.